backend/src/app.js

```diff
@@ -29,7 +29,7 @@ const {
     overrideConsoleErrorToAddSentryCapture
 } = require('./utils');
 
-const nunjucksEnv = new nunjucks.Environment();
+const nunjucksEnv = new nunjucks.configure(config.get('template_path'), {watch: true});
```

Comment:
@abordin-spacefill **issue**: The problem of Jest that force us using `--forceExit` is because of this `watch` can you remove the `watch` or disable it in the unit test and remove the `--forceExit` on jest? 🙏

Reply:
@Its-Alex I made some changes to use `NODE_ENV` to define if we enable the watch on the templates or not. 

Please tell me if it's ok for you now ?